package.json

```diff
@@ -12,6 +12,7 @@
   "scripts": {
     "lint": "conventional-changelog-lint --from=master && coffeelint src",
     "build": "coffee -b -c -o lib/ src/",
+    "watch": "coffee -w -b -c -o lib/ src/",
```

Comment:
I was reading https://github.com/apiaryio/dredd/issues/867 before this one, hence my comment there. The only important part is that I'd suggest the name of the command to be `build:watch` instead of `watch`.